be/src/exec/schema_scanner/schema_be_tablets_scanner.cpp

```diff
@@ -34,6 +34,7 @@ SchemaScanner::ColumnDesc SchemaBeTabletsScanner::_s_columns[] = {
         {"MAX_VERSION", TYPE_BIGINT, sizeof(int64_t), false},
         {"MIN_VERSION", TYPE_BIGINT, sizeof(int64_t), false},
         {"NUM_ROWSET", TYPE_BIGINT, sizeof(int64_t), false},
+        {"NUM_SEGMENT", TYPE_BIGINT, sizeof(int64_t), false},
```

Comment:
specific reason to insert the new column in the middle instead of appending to the end?

Reply:
I just want to put it after `num_rowset` for taking a look more convenient.

Reply:
may not be a good idea from compatibility point of view. Append new columns to the end to get a better compatibility, more friendly to plain text parsers